server/src/metadata/object-metadata/hooks/before-create-one-object.hook.ts

```diff
@@ -1,11 +1,26 @@
-import { Injectable, UnauthorizedException } from '@nestjs/common';
+import {
+  ForbiddenException,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
 
 import {
   BeforeCreateOneHook,
   CreateOneInputType,
 } from '@ptc-org/nestjs-query-graphql';
 
 import { CreateObjectInput } from 'src/metadata/object-metadata/dtos/create-object.input';
+import { standardObjectsNames } from 'src/workspace/workspace-manager/standard-objects/standard-object-metadata';
+
+const OBJECT_TYPES = [
+  'featureFlag',
+  'refreshToken',
+  'UserWorkspaceMemberName',
+  'UserWorkspaceMember',
+  'Workspace',
+  'User',
+  ...standardObjectsNames,
```

Comment:
This part (standardObjectsNames) should not be necessary because we already have an unicity constraint on the nameSingular/namePlural in DB and standard objects should already exist at that time. 
Later we can add a different validation check if we want an extra layer (above postgres) but it should have its own error message imho